man/signal-cli-dbus.3.adoc

```diff
@@ -0,0 +1,205 @@
+/////
+vim:set ts=4 sw=4 tw=82 noet:
+/////
+:quotes.~:
+
+= signal-cli-dbus (3)
```

Comment:
I think 3 is not the best place for it, it's for programming libraries.
Most dbus descriptions on my system are in 5.